crates/ty_python_semantic/resources/mdtest/annotations/callable.md

```diff
@@ -42,6 +42,7 @@ Or, when it's a literal type:
 
 ```py
 # error: [invalid-type-form] "The first argument to `Callable` must be either a list of types, ParamSpec, Concatenate, or `...`"
+# error: [invalid-type-form] "Int literals are not allowed in this context in a type expression"
```

Comment:
I think _ideally_ the first error here is sufficient and we don't need the second one, too. But if that's really difficult, it's not a big deal.